lib/index.js

```diff
@@ -1167,22 +1167,38 @@ module.exports = (function() {
             }
             return ab;
         };
-        substitution = self.imageToBuffer(substitution)
+        try {
+            substitution = self.imageToBuffer(substitution)
+        } catch (error) {
+            console.error("Error on reading image object : ", error)
+            return false;
+        }
+        if (substitution == null) {
+            return false;
+        }
+        
         // put image to media.
         self.archive.file('xl/media/image' + maxFildId + '.jpg', toArrayBuffer(substitution), {binary:true, base64:false});
-        var dimension = sizeOf(substitution);
+        try {
+            var dimension = sizeOf(substitution);
+        } catch (error) {
```

Comment:
Same as above.
I do not think that we should hide underlying error from the develop.
He should understand that `substitution` is broken in specific way and fix that in his code.